inference-engine/src/preprocessing/ie_preprocess_gapi_kernels.cpp

```diff
@@ -738,714 +738,366 @@ struct typed_resizeLinearU8C3C4 {
+        alpha_type *alphaw = &alpha[w * maxdif];
+        for (int i = 0; i < maxdif; i++) {
+            if (i + i0 == index0) {
+                alphaw[i] = xmaps[w].alpha0;
+
+            } else if (i + i0 == index1 - 1) {
+                alphaw[i] = xmaps[w].alpha1;
+
+            } else if (i + i0 > index0 && i + i0 < index1 - 1) {
+                alphaw[i] = mapper.alpha;
+
+            } else {
+                alphaw[i] = 0;
+            }
+        }
+
+        // start input chunk with adjusted position
+        index[w] = i0;
     }
 
-    callRowFunc<KT, LT>(dst, src0, src1, alpha, clone, mapsx, beta, tmp, inSz, outSz,
-                        lpi, length, out.meta().depth);
+    *maxdf = maxdif;
 }
 
-GAPI_FLUID_KERNEL(FScalePlane8u, ScalePlane8u, true) {
-    static const int Window = 1;
-    static const int LPI = 4;
-    static const auto Kind = cv::GFluidKernel::Kind::Resize;
+#if USE_CVKL
+static int getResizeAreaTabSize(int dst_go, int ssize, int dsize, float scale) {
+    static const float threshold = 1e-3f;
+    int max_count = 0;
 
-    static void initScratch(const cv::GMatDesc & in,
-                            Size outSz, int /*interp*/,
-                            cv::gapi::fluid::Buffer & scratch) {
-        initScratchLinear<uchar, linear::Mapper>(in, outSz, scratch, LPI);
-    }
+    for (int col = dst_go; col < dst_go + dsize; col++) {
+        int count = 0;
 
-    static void resetScratch(cv::gapi::fluid::Buffer& /*scratch*/) {
-    }
+        float fsx1 = col * scale;
+        float fsx2 = fsx1 + scale;
 
-    static void run(const cv::gapi::fluid::View & in, Size /*sz*/, int /*interp*/,
-                    cv::gapi::fluid::Buffer & out, cv::gapi::fluid::Buffer & scratch) {
-        calcRowLinear<uint8_t, linear::Mapper,
-                      typed_resizeLinearU8C1<isa_tag_t, linear::Mapper>,
-                      resizeLinearU8C1_suptypes>(in, out, scratch);
-    }
-};
+        int sx1 = static_cast<int>(ceil(fsx1));
+        int sx2 = static_cast<int>(floor(fsx2));
 
-GAPI_FLUID_KERNEL(FScalePlane32f, ScalePlane32f, true) {
-    static const int Window = 1;
-    static const int LPI = 4;
-    static const auto Kind = cv::GFluidKernel::Kind::Resize;
+        sx2 = (std::min)(sx2, ssize - 1);
+        sx1 = (std::min)(sx1, sx2);
 
-    static void initScratch(const cv::GMatDesc & in,
-                            Size outSz, int /*interp*/,
-                            cv::gapi::fluid::Buffer & scratch) {
-        GAPI_DbgAssert(in.depth == CV_32F && in.chan == 1);
+        if (sx1 - fsx1 > threshold) {
+            count++;
+        }
 
-        initScratchLinear<float, linear32f::Mapper>(in, outSz, scratch, 0);
-    }
+        for (int sx = sx1; sx < sx2; sx++) {
+            count++;
+        }
 
-    static void resetScratch(cv::gapi::fluid::Buffer& /*scratch*/) {
+        if (fsx2 - sx2 > threshold) {
+            count++;
+        }
+        max_count = (std::max)(max_count, count);
```

Comment:
I wondering why is not `std::max(max_count, count);`?

Reply:
Done.